CHANGELOG.md

```diff
@@ -2,7 +2,8 @@
 
 ## Master (Unreleased)
 
-- Support correcting `assert_nil` and `refute_nil` in `RSpec/Rails/MinitestAssertions`. ([@G-Rath])
+- Add support `assert_empty`, `assert_not_empty` and `refute_empty` for `RSpec/Rails/MinitestAssertions`. ([@ydah])
+- Support correcting `*_includes` assertions in `RSpec/Rails/MinitestAssertions`. ([@G-Rath])
 - Support correcting `assert_not_equal` and `assert_not_equal` in `RSpec/Rails/MinitestAssertions`. ([@G-Rath])
```

Comment:
I’ve just noticed a duplicate matcher name here

Reply:
🤦 I'll just fix that in this if you're ok with that